test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -43,4 +43,16 @@ private void assertAreUnique(Object... objects) {
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+
+    @Test
+    public void isAnyNull() throws Exception {
+
+        assertTrue(Utils.isAnyNull(null, null));
+
+        assertTrue(Utils.isAnyNull(1, null));
+
+        assertFalse(Utils.isAnyNull(1, 2, 3));
+
+    }
```

Comment:
You may want to consider the case of an empty list :)

On a related note, consider segregating the different methods into its respective categories e.g.
```
// Empty list
....

// Non-empty list
...
```
It would improve readability for yourself and others should there be a need to include more test cases in the future :)

Reply:
Noted. Did not consider lists for this test. How about for Maps and Sets? Seems like it would be worthwhile to do for those too.

Thanks for the feedback!

Reply:
Feel free to include them if you would like but it is not necessary for this PR :) 

This PR serves as an exercise for students to learn the JUnit mechanics, so the most important thing is that you takeaway the knowledge gained and apply them in your project 😄